src/argrelay/custom_integ_res/check_env.bash

```diff
@@ -160,9 +160,9 @@ echo -e "${success_color}INFO:${reset_style} ${field_color}argrelay_version:${re
 
 ########################################################################################################################
 # Report server URL:
-server_host_name="$( jq --raw-output ".connection_config.server_host_name" "${argrelay_dir}/conf/argrelay_client.json" )"
-server_port_number="$( jq --raw-output ".connection_config.server_port_number" "${argrelay_dir}/conf/argrelay_client.json" )"
-echo -e "${success_color}INFO:${reset_style} ${field_color}server_url:${reset_style} http://${server_host_name}:${server_port_number}"
+server_host_name="$( jq --raw-output ".redundant_connections[0].server_host_name" "${argrelay_dir}/conf/argrelay_client.json" )"
+server_port_number="$( jq --raw-output ".redundant_connections[0].server_port_number" "${argrelay_dir}/conf/argrelay_client.json" )"
+echo -e "${success_color}INFO:${reset_style} ${field_color}server_url[0]:${reset_style} http://${server_host_name}:${server_port_number}"
```

Comment:
Same [as here](https://github.com/argrelay/argrelay/pull/101/files#r1685399721).

This can be reported via Python.